### PR TITLE
Revert to OpenSBI v0.8 to test if sel4bench still works

### PR DESCRIPTION
**Pull request by Ivan-Velickovic**
Locally I tested v0.8 with OpenSBI and sel4bench still fails which makes me think that something else is going on. This is a dummy PR to test that change to see if this can be reproduced on the CI.

### CODE REVIEW

**Comment by lsf37** [comment]
Oh, that would be interesting. I was pretty sure I had tracked down the PR, but I might have gotten something wrong.

**Comment by Ivan-Velickovic** [comment]
> Oh, that would be interesting. I was pretty sure I had tracked down the PR, but I might have gotten something wrong.

It might be the change to `generic` OpenSBI platform, before it was HiFive Unleashed specific.

**Comment by Ivan-Velickovic** [comment]
Based on the CI run now, it looks like to me that https://github.com/seL4/seL4/commit/9b039f8e3729a466bfe151a9f24057f551d3ab4e is the culprit. Not sure why though.

File: default.xml [hunk 1]
```diff
@@ -13,7 +13,7 @@
   
   <project name="musllibc.git" path="projects/musllibc" revision="3d6b939e8f05cb1d2a1a8c8166609bf2e652e975" upstream="sel4" dest-branch="sel4"/>
   <project name="nanopb" remote="nanopb" revision="1466e6f953835b191a7f5acf0c06c941d4cd33d9" upstream="master" dest-branch="refs/tags/0.4.3"/>
-  <project name="opensbi" path="tools/opensbi" remote="opensbi" revision="a98258d0b537a295f517bbc8d813007336731fa9" upstream="refs/tags/v0.9" dest-branch="refs/tags/v0.9"/>
+  <project name="opensbi" path="tools/opensbi" remote="opensbi" revision="refs/tags/v0.8"/>
```
Comment (kent-mcleod) [inline default.xml:16]: This needs to update master.xml instead of default.xml

**Pull request closed without merging**